src/start/server.ts

```diff
@@ -77,6 +78,19 @@ async function main() {
     startAsyncMigrations(shutdownSignal.nextGroup.signal)
   }
 
+  // Cluster information
+  await Cluster.init(shutdownSignal.nextGroup.signal)
+
+  Cluster.on('change', (data) => {
+    logger.info('[Cluster] Cluster size changed', {
+      type: 'cluster',
```

Comment:
same here... I think this should probably be `type: 'ECS'` and maybe get it from `data.clusterType` or something